### DISCUSSION TITLE
Release notes for 2.12.0 only showing changes since 2.12.0-rc5, not since 2.11.x

### DISCUSSION THREAD

**Discussion by kastl-ars** [argoproj/argo-cd#19390] [General]
Would it be possible to have the release notes on big updates (like 2.12.0) show the changes not only since the last RC release on this line? Most people using 2.11.x are more interested in what changed since the current 2.11 release. And having to go through six releases to collect all changes is... tedious.

Example:
https://github.com/argoproj/argo-cd/releases/tag/v2.12.0 only shows the changes since 2.12.0-rc5, not since 2.11.x.

Also, the full changelog is only compared to the last RC:
`Full Changelog: https://github.com/argoproj/argo-cd/compare/v2.12.0-rc5...v2.12.0`

Kind Regards,
Johannes

**Comment by crenshaw-dev** [comment]
The problem is based on the current behavior of goreleaser's change log generation. I created an issue to track fixing it: https://github.com/argoproj/argo-cd/issues/19443

**Reply by kastl-ars** [reply]
Thanks!
